src/main/kotlin/mate/academy/model/IsoscelesTrapezoid.kt

```diff
@@ -0,0 +1,26 @@
+package mate.academy.model
+
+private const val DIVIDER = 2.0
+
+class IsoscelesTrapezoid(
+    color: Color, val firstBase: Int,
+    val secondBase: Int, val height: Int

```

Comment:
please keep extra line for field, so it will be easier to read your code
```suggestion
    color: Color, 
    val firstBase: Int,
    val secondBase: Int, 
    val height: Int
```